tests/test_signals.py

```diff
@@ -0,0 +1,46 @@
+from cms.api import create_page
+from cms.test_utils.testcases import CMSTestCase
+from cms.test_utils.util.context_managers import signal_tester
+
+from djangocms_versioning import constants, signals
+from djangocms_versioning.models import Version
+
+from djangocms_content_expiry.models import ContentExpiry
+
+
+class ContentExpirySignalTestCase(CMSTestCase):
+    def test_content_expiry_creation_signal(self):
+        """
+        Test content expiry exists in the new version
```

Comment:
Starting with test, or test that is obvious as we are in a test, we should be more direct.

```suggestion
       Content expiry should exist in new versions, always.
```